custom/icds/location_reassignment/processor.py

```diff
@@ -24,7 +25,11 @@ def process(self):
                 self._perform_transitions(operation, transitions)
 
     def _perform_transitions(self, operation, transitions):
-        for old_site_codes, new_site_codes in transitions.items():
+        for from_site_codes, to_site_codes in transitions.items():
```

Comment:
This is going to be confusing to devs unfamiliar with this code, could you add a comment?

Reply:
agreed, it confuses me at times too, this definitely needs more than a comment.
i am thinking that this logic should be at one place only. I will add a comment for now.

Reply:
Cool, agreed that getting that logic into one place would be great.